editor/edit-post/layout/style.scss

```diff
@@ -47,6 +54,11 @@
 
 .editor-layout__content {
 	position: relative;
-	overflow-y: auto;
-	-webkit-overflow-scrolling: touch;
+
+	// on mobile the main content area has to scroll
```

Comment:
Is this targeting iOS specifically? Does it impact larger-display iOS like iPads? Is the `-webkit-overflow-scrolling` style unnecessary if we're intending to exclude it?

Reply:
I'm essentially normalizing the scrolling behavior across the mobile breakpoint, yes. And though the iPad also suffers from the same scrolling issue, the larger screen real estate makes the scroll issue less annoying, so it'd be good to keep the property there.